src/sympc/tensor/mpc_tensor.py

```diff
@@ -685,6 +685,19 @@ def __apply_private_op(
             else:
                 raise TypeError("Invalid Share Class")
 
+        elif op_str == "xor":
+            if self.session.ring_size == 2:
+                op = getattr(operator, op_str)
+                shares = [
+                    op(*share_tuple)
+                    for share_tuple in zip(self.share_ptrs, y.share_ptrs)
+                ]
+                result = MPCTensor(
+                    shares=shares, shape=self.shape, session=self.session
+                )
+            else:
+                return self + y - (self * y * 2)
```

Comment:
This also requires ring size 2 right?

Reply:
Oh for case of prime numbers?

Reply:
@aanurraj ^^

Reply:
Yeah it works ,when the secret is shares of a bit, the above formula.